themes/default/content/docs/intro/concepts/state.md

```diff
@@ -220,13 +227,13 @@ $ pulumi login gs://<my-pulumi-state-bucket>
 
 To configure credentials for this backend, see [Application Default Credentials](https://cloud.google.com/docs/authentication/production). For additional configuration options, see [GCP Setup]({{< relref "/registry/packages/gcp/installation-configuration" >}}). If you're new to Google Cloud Storage, see [the Google Cloud documentation](https://cloud.google.com/storage/docs/quickstarts).
 
-### Logging Out
+## Logging Out
```

Comment:
can we include this as part of the logging in section instead just at the end of that section?

Reply:
Yeah, that makes sense.